mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsFragment.java

```diff
@@ -550,10 +551,10 @@ public void showClientInformation(final Client client) {
             if (client.isImagePresent()) {
                 loadClientProfileImage();
             } else {
-                iv_clientImage.setImageDrawable(
-                        ResourcesCompat.getDrawable(getResources(), R.drawable
-                                .ic_launcher, null));
-
+                iv_clientImage.setImageBitmap(AvatarGenerator.Companion.avatarImage(
+                        getActivity().getBaseContext(), 75,
+                        AvatarConstants.Companion.getCIRCLE(),
+                        client.getDisplayName()).getBitmap());
```

Comment:
To make the code more readable try to declare and segregate things.

Reply:
I will take care of it :)